python/fichas/listas.py

```diff
@@ -75,7 +75,7 @@ def is_elem(elem, values: list) -> bool:
     return False
 
 
-def maximum(values: list[int]) -> int:
+def maximum(values: list) -> int:
```

Comment:
```suggestion
def maximum(values: list[int]) -> int:
```

Since we are returning an int, it means the list should be a list of ints.